content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx

```diff
@@ -147,6 +147,30 @@ const xprisma = prisma.$extends({
 })
 ```
 
+### Mutate the result of a query
```

Comment:
```suggestion
## Mutate the result of a query
```

Reply:
Isn't this an examples as the others before it?

Reply:
Yep, good point - I'll close this suggestion